Mathlib/FieldTheory/NormalClosure.lean

```diff
@@ -10,19 +10,32 @@ import Mathlib.Order.Closure
 #align_import field_theory.normal from "leanprover-community/mathlib"@"9fb8964792b4237dac6200193a0d533f1b3f7423"
 /-!
 # Normal closures
-The normal closure of a tower of fields `L/K/F` is the smallest intermediate field of `L/K` that
-contains the image of every `F`-algebra embedding `K →ₐ[F] L`.
-
-## Main Definitions
-- `normalClosure F K L` a tower of fields `L/K/F`.
+The normal closure `normalClosure F K L` of a field extension `K/F`
```

Comment:
Can you add here a section "Main definitions" with `IsNormalClosure` and `normalClosure`? Thanks!

Reply:
I removed the "main definitions" section in favor of more prose-style exposition in which I explain the two main definitions, but I can also restore the stricter format.